src/d/actor/d_a_branch.cpp

```diff
@@ -3,17 +3,58 @@
 // Translation Unit: d_a_branch.cpp
 //
 
-#include "d_a_branch.h"
+#include "JSystem/JKernel/JKRHeap.h"
+#include "JSystem/JUtility/JUTAssert.h"
+#include "f_op/f_op_actor.h"
+#include "f_op/f_op_actor_mng.h"
+#include "d/d_procname.h"
+#include "m_Do/m_Do_ext.h"
+#include "d/d_com_inf_game.h"
+#include "d/d_resorce.h"
 #include "dolphin/types.h"
 
+
+class daBranch_c : public fopAc_ac_c {
+public:
+    void set_mtx();
+    void set_anim(int, int, int);
+    void demoPlay(mDoExt_McaMorf*);
+
+    s32 CreateHeap();
+    static s32 solidHeapCB(fopAc_ac_c*);
+
+    request_of_phase_process_class mPhs;
+    s8 dummy[0x10];
+    mDoExt_McaMorf* mAnims[2];
+
+    static char m_arcname[];
+};
+
+char daBranch_c::m_arcname[] = "Kwood_00";
+
 /* 00000078-00000128       .text set_mtx__10daBranch_cFv */
 void daBranch_c::set_mtx() {
-    /* Nonmatching */
+    //for (int i = 0; i < 2; i ++) {
+
+   // }
```

Comment:
is this still nonmatching? if so the comment should probably be preserved

Reply:
Added